101-vm-with-standardssd-disk/azuredeploy.json

```diff
@@ -0,0 +1,249 @@
+          "apiVersion": "[variables('apiVersion')]",
+          "type": "Microsoft.Network/networkInterfaces",
+          "name": "[variables('networkInterfaceName')]",
+          "location": "[parameters('location')]",
+          "dependsOn": [
+              "[concat('Microsoft.Network/publicIPAddresses/', variables('publicIpAddressName'))]",
+              "[concat('Microsoft.Network/virtualNetworks/', parameters('virtualNetworkName'))]"
+          ],
+          "properties": {
+              "ipConfigurations": [
+                  {
+                      "name": "ipconfig1",
+                      "properties": {
+                          "privateIPAllocationMethod": "Dynamic",
+                          "publicIPAddress": {
+                              "id": "[resourceId('Microsoft.Network/publicIPAddresses',variables('publicIpAddressName'))]"
+                          },
+                          "subnet": {
+                              "id": "[variables('subnetRef')]"
+                          }
+                      }
+                  }
+              ]
+          }
+      },
+      {
+          "apiVersion": "[variables('computeResouresApiVersion')]",
+          "type": "Microsoft.Compute/virtualMachines",
+          "name": "[parameters('virtualMachineName')]",
+          "location": "[parameters('location')]",
+          "dependsOn": [
+              "[concat('Microsoft.Network/networkInterfaces/', variables('networkInterfaceName'))]"
+          ],
+          "properties": {
+              "hardwareProfile": {
+                  "vmSize": "[parameters('virtualMachineSize')]"
+              },
+              "osProfile": {
+
+                  "computername": "[parameters('virtualMachineName')]",
+
+                  "adminUsername": "[parameters('adminUsername')]",
+
+                  "adminPassword": "[parameters('adminPassword')]"
+
+              },
+
+              "storageProfile":
+
+          {
+              
+                      
+               "osDisk": {
+                      "osType": "Windows",
+                      "name": "[variables('OSDiskName')]",
+                      "caching": "ReadWrite",
+                      "createOption": "FromImage",
+                       "managedDisk": {
+                          "storageAccountType": "[parameters('diskType')]"
+},  
+                          
+              "diskSizeGB": "128"
+                  },
+              "imageReference":
+
+              {
+
+                  "publisher": "[variables('imagePublisher')]",
+
+                  "offer": "[variables('imageOffer')]",
+
+                  "sku": "[parameters('windowsOSVersion')]",
+
+                  "version": "latest"
+
+              },
+              "copy": [
+          {
+            "name": "dataDisks",
+            "count": "[parameters('DataDisksCount')]",
+            "input": {
+              "name": "[concat(parameters('virtualMachineName'),'DataDisk',CopyIndex('dataDisks'))]",
```

Comment:
CopyIndex should be copyIndex (fix casing)

Reply:
Done